edx_sysadmin/management/commands/tests/test_git_add_course.py

```diff
@@ -1,20 +1,19 @@
 """
 Provide tests for git_add_course management command.
 """
+# pylint: disable=wrong-import-order
 import logging
 import os
 import shutil
 import subprocess
 from uuid import uuid4
 
+import six
```

Comment:
this is an out of scope question, but I'm curious: do we still need six? I don't think we're supporting python 2 any more.

Reply:
I agree, we should probably now replace it with `io` and remove the six and its usages. e.g. 

`from io import StringIO` instead of `from six import StringIO`.